tools/rna_tools/cmv/hmmcv.xml

```diff
@@ -13,7 +13,8 @@
         @COMMON_ARGS@
         -m $model_filepath
         -r $hmmcompare_file
-        -p ./
+        -p  `pwd`
+        > 'warnings'
```

Comment:
wenn du nach warnings pipest wird alles in ein file namens `warnings` geschrieben und der user sieht es nicht korrekt?